src/Symfony/Bundle/WebProfilerBundle/EventListener/WebDebugToolbarListener.php

```diff
@@ -98,6 +98,7 @@ public function onKernelResponse(FilterResponseEvent $event)
             || $response->isRedirection()
             || ($response->headers->has('Content-Type') && false === strpos($response->headers->get('Content-Type'), 'html'))
             || 'html' !== $request->getRequestFormat()
+            || false !== strpos($response->headers->get('Content-Disposition'), 'attachment')
```

Comment:
The content-disposition types are case-insensitive according to https://tools.ietf.org/html/rfc2183
Also this current check will also trigger for `Content-Disposition: <something-else>; filename=attachment` which is probably not intended.